src/Bicep.Cli.IntegrationTests/DecompileTests.cs

```diff
@@ -129,10 +116,15 @@ public void Decompilation_of_empty_template_succeeds()
             bicepFile.Should().BeEquivalentTo("");
         }
 
+        [TestMethod]
+        public void Decompilation_of_nested_template_creates_module_file()
+        {
```

Comment:
Is this PR still in draft-ish state?

Reply:
Sometimes you just have to test the test framework, to keep it on its toes...

Reply:
Mstest does indeed behave in suspiciously mysterious ways sometimes.